### DISCUSSION TITLE
希望提高部分文字的字重

### DISCUSSION THREAD

**Discussion by chiyuki0325** [Qexo/Qexo#142] [Ideas]
![图片](https://user-images.githubusercontent.com/73830635/166448763-4c9d574c-fd6c-4820-8741-16035f322dcc.png)
![图片](https://user-images.githubusercontent.com/73830635/166449331-860c7ac3-a828-47d3-9dda-37364fea11d4.png)

在 1920x1080 的显示器上，预览和部分提示文本字体过细，对于非微软雅黑字体，部分地方甚至显示不清。希望把原 300 字重的地方改为 400 或 500 字重。

**Comment by am-abudu** [comment]
6a67d6be79138d7f3b85a5f25a655539956fbcc9
